test/regress/cli/regress1/parse-skolem-test-array-deq.smt2

```diff
@@ -0,0 +1,8 @@
+(set-option :parse-skolem-definitions true)
```

Comment:
You will need to add these 3 files to https://github.com/cvc5/cvc5/blob/main/test/regress/cli/CMakeLists.txt.

You will also need to add an expected output and command line options, e.g. I think this benchmark should have the lines at the top of this file:
```
; COMMAND-LINE: --parse-skolem-definitions --print-skolem-definitions
; EXPECT: unsat